app/registration/views/patient-hiv-test-input.html

```diff
@@ -0,0 +1,22 @@
+<div class="panel panel-primary content-border-margin"><!-- address -->
+    <div class="panel-heading">
+        <h3 class="panel-title">{{'PATIENT_INFO_HIV_TEST' | translate}}</h3>
+    </div>
+    <div class="panel-body">
+        <div ng-init="attributeRow = patientCommon.filterPersonAttributesForCurrStep(patientConfiguration.customAttributeRows(), additionalPatientAttributes.testing)">
```

Comment:
Looks like ```attributeRow``` is only used in the ```ng-repeat``` so there is no need to have it in the entire ```$scope``` . 

```patientCommon.filterPersonAttributesForCurrStep()``` should be called in ```ng-repeat```.

This is because our styleguide recomends using ```controllerAs``` with ```vm```, code executed inside```ng-init``` is not testable and ```ng-int``` is also not available is angular 2+.